developer-docs-site/docs/nodes/validator-node/operator/connect-to-aptos-network.md

```diff
@@ -268,6 +268,41 @@ After your validator node joined the validator set, you can verify the correctne
     
     You should expect the active value for your `StakePool` to keep increasing. It is updated at every epoch.
 
+## Seeing your stake pool information
+
+To see the details of your stake pool, run the below CLI command with the `get-stake-pool` option by providing the `--owner-address` and `--url` fields. This example is for Premainnet. For other networks, use the appropriate REST URL for the `--url` field. See [Aptos Blockchain Deployments](/nodes/aptos-deployments) for `--url` field values for other networks. 
+
+The below command is for an example owner address `0x0756c80f0597fc221fe043d5388949b34151a4efe5753965bbfb0ed7d0be08ea` and it will display the following information on the owner's stake pool on the premainnet:
+
+- Status of the stake: pending active, active, pending-inactive, inactive.
```

Comment:
I added an example output in a comment below. Can you go through them and add the new fields here + documentation? 🙏

Reply:
Yes yes, thanks! Let me work these in and push an update.